install/mysql/glpi-empty.sql

```diff
@@ -2785,9 +2785,9 @@ CREATE TABLE `glpi_entities` (
   `contracts_id_default` int unsigned NOT NULL DEFAULT '0',
   `enable_custom_css` int NOT NULL DEFAULT '-2',
   `custom_css_code` text,
-  `latitude` varchar(255) DEFAULT NULL,
-  `longitude` varchar(255) DEFAULT NULL,
-  `altitude` varchar(255) DEFAULT NULL,
+  `latitude` decimal(9,6) NOT NULL DEFAULT '0.0000',
+  `longitude` decimal(9,6) NOT NULL DEFAULT '0.0000',
+  `altitude` decimal(9,6) NOT NULL DEFAULT '0.0000',
```

Comment:
Why did you added the `NOT NULL` option? It is probably preferable to keep a null value when coordinates are not defined.

```suggestion
  `latitude` decimal(9,6) DEFAULT NULL,
  `longitude` decimal(9,6) DEFAULT NULL,
  `altitude` decimal(9,6) DEFAULT NULL,
```

Reply:
Sorry for the delay. I changed the fields to be NULL by default.